interface.html

```diff
@@ -70,6 +70,41 @@
         </div>
 
         <div class="title-header">
+          <p>Chat options</p>
+        </div>
+        <div class="form-group clearfix inline-radio">
+          <div class="col-sm-4 control-label">
+            <label>Do you want to allow users to chat?</label>
+            <p>Selecting "Yes" will place a button in the detail view, that users can tap on to start a conversation.</p>
```

Comment:
*Selecting "Yes" will place a **Send message** button in the user profile. Tapping on the button will start a conversation.*